File: app/src/main/java/dev/ragnarok/fenrir/api/interfaces/IStoreApi.java
package dev.ragnarok.fenrir.api.interfaces;

import dev.ragnarok.fenrir.api.model.VkApiStickerSetsData;
import dev.ragnarok.fenrir.api.model.VkApiStickersKeywords;
import io.reactivex.rxjava3.core.Single;


public interface IStoreApi {
    Single<VkApiStickersKeywords> getStickerKeywords();

    Single<VkApiStickerSetsData> getStickers();
}

File: app/src/main/java/dev/ragnarok/fenrir/api/interfaces/IStoreApi.kt
package dev.ragnarok.fenrir.api.interfaces

import dev.ragnarok.fenrir.api.model.VKApiStickerSetsData
import dev.ragnarok.fenrir.api.model.VKApiStickersKeywords
import io.reactivex.rxjava3.core.Single

interface IStoreApi {
    val stickerKeywords: Single<VKApiStickersKeywords>
    val stickers: Single<VKApiStickerSetsData>
}